src/sentry/lang/javascript/processor.py

```diff
@@ -52,7 +52,7 @@ class ZeroReturnError(Exception):
 """, re.X | re.I)
 VERSION_RE = re.compile(r'^[a-f0-9]{32}|[a-f0-9]{40}$', re.I)
 NODE_MODULES_RE = re.compile(r'\bnode_modules/')
-
+SOURCE_MAPPING_URL_RE = re.compile(r'\/\/(?:#|@) ?sourceMappingURL=(.*)$', re.I)
```

Comment:
Should the space after `//#` be optional? We don't allow it to be optional elsewhere.